### ISSUE TITLE
This program kills the VA Process.

### ISSUE THREAD

**Issue by Jay8454** [EDCD/EDDI#2646]
This program kills the VA Process. I have the Steam version of VA (Paid). I verified that EDDI was installed into the VA directory apps folder. VA will no longer start up completely. However, I did allow the EDDI to update, and this has changed. I now get a startup, and EDDI is listed in the settings/plugin manager along with EliteVA, HCS, and EDDI 4.1.0-b3. However, EDDI is ticked but not selectable as an "option." So, nothing is showing in the VA options list, only the other plug-ins.

**Comment by Tkael** [comment]
Hi there and welcome! If VoiceAttack is crashing then please forward `VoiceAttackFault.txt` in the VoiceAttack install directory and `eddi.log` in the `%appdata%/EDDI` directory so that I can help troubleshoot any issues you are encountering?

Did you review <https://github.com/EDCD/EDDI/blob/develop/VoiceAttack-Integration.md>? EDDI.vap (in the EDDI install directory) contains some basic commands for interacting with the UI which should be very helpful.

**Comment by Jay8454** [comment]
Is EDDI supposed to be running while VA is? Cause I don't see the EDDI running even though it says it's running. There is nothing in TaskManager of EDDI running. So, VA is no longer crashing. 
The eddi.log:
`2024-08-29T19:44:11 [Info] EDDI:.ctor EDDI 4.0.4 starting
2024-08-29T19:44:12 [Info] CargoMonitor:.ctor Initialized Cargo monitor
2024-08-29T19:44:12 [Info] CrimeMonitor:initializeCrimeMonitor Initialized Crime monitor
2024-08-29T19:44:12 [Info] MaterialMonitor:.ctor Initialized Material monitor
2024-08-29T19:44:12 [Info] MissionMonitor:initializeMissionMonitor Initialized Mission monitor
2024-08-29T19:44:12 [Info] StarSystemSqLiteRepository:CreateOrUpdateDatabase Starsystem repository schema is version 3
2024-08-29T19:44:12 [Info] EDDNResponder:.ctor Initialized EDDN Responder
2024-08-29T19:44:12 [Info] EDSMResponder:.ctor Initialized EDSM Responder
2024-08-29T19:44:12 [Info] VoiceAttackResponder:.ctor Started VoiceAttack responder
2024-08-29T19:44:13 [Info] NavigationMonitor:.ctor Initialized Navigation monitor
2024-08-29T19:44:13 [Info] ShipMonitor:.ctor Initialized Ship Monitor
2024-08-29T19:44:13 [Info] StatusMonitor:.ctor Initialized Status monitor
2024-08-29T19:44:13 [Info] EDDI:.ctor EDDI 4.0.4 initialised
2024-08-29T19:44:13 [Info] EDDI:Start Starting keepalive for Galnet monitor
2024-08-29T19:44:13 [Info] EDDI:Start Starting keepalive for Journal monitor
2024-08-29T19:44:13 [Info] EDDI:Start Starting keepalive for Mission monitor
2024-08-29T19:44:13 [Info] EDDI:Start Starting keepalive for Status monitor
2024-08-29T19:44:13 [Info] EDDI:Start Started EDDN Responder
2024-08-29T19:44:13 [Info] EDDI:keepAlive Starting Galnet monitor (0)
2024-08-29T19:44:13 [Info] EDDI:keepAlive Starting Status monitor (0)
2024-08-29T19:44:13 [Info] EDDI:Start Started EDSM Responder
2024-08-29T19:44:13 [Info] EDDI:keepAlive Starting Journal monitor (0)
2024-08-29T19:44:13 [Info] EDDI:keepAlive Starting Mission monitor (0)
2024-08-29T19:44:13 [Info] DataProviderService:syncFromStarMapService Syncing all flight logs from EDSM
2024-08-29T19:44:13 [Info] EDDI:Start Started Inara Responder
2024-08-29T19:44:13 [Info] InaraResponder:Start Initialized Inara Responder
2024-08-29T19:44:13 [Info] EDDI:Start Started Speech Responder
2024-08-29T19:44:13 [Info] SpeechResponder:Start Initialized Speech Responder
2024-08-29T19:44:13 [Info] EDDI:Start Started VoiceAttack responder
2024-08-29T19:44:13 [Info] JournalMonitor:ParseJournalEntry GameVersion: 4.0.0.1808, Build r304197/r0.
2024-08-29T19:44:13 [Info] JournalMonitor:ParseJournalEntry Active expansions... Horizons: True, Odyssey: True.
2024-08-29T19:44:14 [Info] EDDI:eventLocation Vehicle mode is Ship
2024-08-29T19:44:14 [Info] EDDI:eventLocation Location StarSystem: Kongga
2024-08-29T19:44:14 [Info] EDDI:.ctor EDDI access to the Frontier API is enabled.
2024-08-29T19:44:15 [Warning] DataProviderService:syncFromStarMapService No flight logs received for 443 system(s)..
2024-08-29T19:44:15 [Info] VoiceAttackPlugin:VA_Init1 Initialising EDDI VoiceAttack plugin
2024-08-29T19:44:16 [Info] VoiceAttackPlugin:VA_Init1 EDDI VoiceAttack plugin initialization complete
`The Fault file:
`8/28/2024 2:55:59 PM - Plugin exit exception (EDDI 4.1.0-b3): Exception has been thrown by the target of an invocation.

8/28/2024 3:12:44 PM - Plugin exit exception (EDDI 4.1.0-b3): Exception has been thrown by the target of an invocation.

8/28/2024 3:15:28 PM - Plugin exit exception (EDDI 4.1.0-b3): Exception has been thrown by the target of an invocation.

`

**Comment by nepomuk16321** [comment]
Hi @Jay8454 and sorry for the possibly stupid question. Have you assigned a key combination in VA to start the EDDI GUI ?

**Comment by Jay8454** [comment]
That's a great question! Even simple things can sometimes be overlooked. In this case, the answer is no, not key combo assigned there's a GUI?
Thank you for the reply.

**Comment by nepomuk16321** [comment]
Surely you have already started EDDI without VA (i.e. stand alone)? I mean this interface. 
Perhaps we should switch to the discussion area.

**Comment by Jay8454** [comment]
Sure. Discord?

**Comment by nepomuk16321** [comment]
sorry, I'm not on discord and my English is too bad. I often use deepL
https://github.com/EDCD/EDDI/discussions/2648

**Comment by Jay8454** [comment]
So, the key comb method doesn't work however atl+e does. Ok okay it works with shif,t ctrl, alt, and, enter.

**Comment by Tkael** [comment]
If EDDI is installed in the VoiceAttack Apps directory then it will run as a plugin within VoiceAttack and it will not have a separate entry in the Task Manager. You should see a line in VoiceAttack like `Plugin EDDI 4.0.4 initialized` which will confirm that EDDI is running.

EDDI does have a UI that you can access while VoiceAttack is running. The commands in EDDI.vap can be used to open and manipulate the UI. Once EDDI.vap is loaded and selected then you should be able to open the UI by saying either 'Open EDDI' or 'Configure EDDI'. Please see https://github.com/EDCD/EDDI/blob/develop/VoiceAttack-Integration.md#eddis-voiceattack-profile for details.

As expressed above, you can also assign key combos to interact with the UI if you wish using the commands provided in EDDI.vap.

**Comment by Tkael** [comment]
Closing as the OP issue (of VA crashing) appears to be resolved and the secondary issue (of accessing EDDI's UI from within VoiceAttack) has been explained. o7

**Comment by Jay8454** [comment]
Despite my efforts, I've found both EDDI and HCS to be incompatible with my needs. While I believe AI integration could improve EDDI's functionality, I've encountered significant challenges with the software's current state.

I've also experimented with simpler solutions, like EDDiscovery, which aligns more closely with my preference for a straightforward approach. Given the complexities of these programs and my desire to focus on enjoying the game, I've decided to stick with EDCoPilot and EDDiscovery.
